.buildkite/commands/lint-localized-strings-format.sh

```diff
@@ -1,43 +1,7 @@
 #!/bin/bash -eu
 
-echo "--- :rubygems: Setting up Gems"
-install_gems
-
-echo "--- :cocoapods: Setting up Pods"
-install_cocoapods
-
 echo "--- :writing_hand: Copy Files"
 mkdir -pv ~/.configure/wordpress-ios/secrets
 cp -v fastlane/env/project.env-example ~/.configure/wordpress-ios/secrets/project.env
```

Comment:
I wonder whether we would benefit from removing the `before_all` check for this env file and instead rely on the lanes that need secret values to check for their availability.

https://github.com/wordpress-mobile/WordPress-iOS/blob/adbb17ef1af5c0e0e109938c7b24aa706fe0ba79/fastlane/Fastfile#L124-L126

This is the first time the global check bothers me. I guess that's because it's the first time I need to setup the file only to satisfy the check, not because the code I run needs a value from it.

Reply:
I actually wondered about this in the past too.

Probably for another PR, and even a small "project" to not only change this on all repos, but also probably get rid of the requirement for `~/.*-env.default` files too (which have been bothering me too, not only because those files clutters my `$HOME`, but also because I don't use those in practice, as I put all my tokens and the likes that we expect in those files… in my `~/.zshrc` anyway (since they are mostly the same values for all repos, and I prefer having a single source of truth than one per project)

Reply:
+1

We could devise a system that checks the environment for all the required tokens, then delegate to each user how to provide them.

Reply:
It happens that I also just talked about a similar topic with @rynaardb minutes ago, because putting all my tokens in `~/.zshrc` like I've been doing will create an issue now that I'll have to do Tumblr releases, which will likely use similar ENV var names for GitHub and Buildkite tokens and the like… but will need different values for it.

My braindump idea to improve on that while avoid polluting `$HOME` so that each user can provide the required tokens, but still be able to have different tokens (and different values for the same ENV vars) for different repos would basically be:

```ruby
local_env_file = File.join(PROJECT_ROOT_FOLDER, 'local.env')
Dotenv.load(local_env_file) if File.exist?(local_env_file)
```
And then add `local.env` to the `.gitignore` (†).

That way users will have the liberty to either not use that `local.env` file at all and just declare their ENV vars in `~/.zshrc` or wherever they prefer… or, if that's more convenient for them (especially for Release Managers who do releases of different apps that require different tokens), create a `local.env` file at the root of the dir where they cloned the repo and use that.

(†) The name of the `local.env` file is inspired from the name of the `local.properties` file used in a similar fashion for Android Projects to define Java properties only for the local user)